src/mesh/gmsh_reader.cpp

```diff
@@ -1147,17 +1146,21 @@ bool get_new_rotated_indices_3D(const dealii::CellAccessor<dim, spacedim>& cell,
     }
 
     if (good_rotation) {
-        pcout << "-- FOUND LAST GOOD ROTATION, NOW EXITING ROTATION ROUTINE --" << std::endl;
+        // pcout << "-- FOUND LAST GOOD ROTATION, NOW EXITING ROTATION ROUTINE --" << std::endl;
```

Comment:
remove commented code or add a parameter `bool gmsh_reader_verbose_output` similar to what's done in ODE solver

Reply:
Fixed this in the recent commit.